src/main/java/cuchaz/enigma/utils/LangUtils.java

```diff
@@ -0,0 +1,92 @@
+package cuchaz.enigma.utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Map;
+
+import com.google.common.collect.Maps;
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+
+import cuchaz.enigma.config.Config;
+
+public class LangUtils {
+	public static final String DEFAULT_LANGUAGE = "en_us";
+	private static final Gson GSON = new Gson();
+	private static Map<String, String> translations = Maps.newHashMap();
+	private static Map<String, String> defaultTranslations = Maps.newHashMap();
+	private static Map<String, String> languageNames = Maps.newHashMap();
+	
+	static {
+		putInCache(Config.getInstance().language);
+		putInCache(DEFAULT_LANGUAGE);
+	}
+	
+	@SuppressWarnings("unchecked")
+	public static void putInCache(String language) {
```

Comment:
I think you should make this method return a map instead (empty map in case of missing/io exception), rename it to `load`, and use it like this:
```java
	static {
		translations = load(Config.getInstance().language);
		defaultTranslations = load(DEFAULT_LANGUAGE);
	}
```